lib/rules/no-test-module-for.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const types = require('../utils/types');
+const emberUtils = require('../utils/ember');
+
+const ERROR_MESSAGE = 'moduleFor* apis are are not allowed.';
```

Comment:
This error message would be more helpful if it suggested the fix too: 

```
Use `module` instead of `moduleFor...`
```

Reply:
changed in 4be2bd8a62e9fc178f7aa0bd0f0e725b3f81bde1